web-wallet/src/lib/containers/StakeContract/StakeContract.svelte

```diff
@@ -47,11 +55,11 @@
     getKey("minAllowedStake"),
   ]);
 
-  /** @type {Record<string, (info: WalletStakeInfo) => boolean>} */
+  /** @type {Record<string, (info: StakeInfo) => boolean>} */
   const disablingConditions = {
-    stake: (info) => info.has_staked,
-    unstake: (info) => !info.has_staked,
-    "withdraw-rewards": (info) => info.reward <= 0,
+    stake: (info) => !!info.amount,
```

Comment:
As I told you this was just a quick update to avoid the UI erroring and these conditions must be checked: but as far as I know if the amount is not `null` you still have a stake (maybe with rewards to claim) even if values are zeroes.